replicator/src/test/scala/com/evolutiongaming/kafka/journal/replicator/BatchSpec.scala

```diff
@@ -41,164 +41,170 @@ class BatchSpec extends AnyFunSuite with Matchers {
           mark(offset   = 4),
         ),
-        List(appends(4, append(offset = 1, seqNr = 1), append(offset = 3, seqNr = 2))),
+        List(appends(3, append(offset = 1, seqNr = 1), append(offset = 3, seqNr = 2))),
       ),
-      (Nel.of(delete(offset = 1, seqNr = 1)), List(deletes(offset = 1, seqNr = 1))),
-      (Nel.of(mark(offset = 1), delete(offset = 2, seqNr = 1)), List(deletes(offset = 2, seqNr = 1))),
-      (Nel.of(delete(offset = 1, seqNr = 1), mark(offset = 2)), List(deletes(offset = 2, seqNr = 1))),
+      (Nel.of(delete(offset = 1, to = 1)), List(deletes(offset = 1, to = 1))),
+      (Nel.of(mark(offset = 1), delete(offset = 2, to = 1)), List(deletes(offset = 2, to = 1))),
+      (Nel.of(delete(offset = 1, to = 1), mark(offset = 2)), List(deletes(offset = 1, to = 1))),
       (
-        Nel.of(delete(offset = 1, seqNr = 1), append(offset = 2, seqNr = 2)),
-        List(deletes(offset = 1, seqNr = 1), appends(2, append(offset = 2, seqNr = 2))),
+        Nel.of(delete(offset = 1, to = 1), append(offset = 2, seqNr = 2)),
+        List(deletes(offset = 1, to = 1), appends(2, append(offset = 2, seqNr = 2))),
       ),
       (
-        Nel.of(append(offset = 1, seqNr = 2), delete(offset = 2, seqNr = 1)),
-        List(appends(1, append(offset = 1, seqNr = 2)), deletes(offset = 2, seqNr = 1)),
+        Nel.of(append(offset = 1, seqNr = 2), delete(offset = 2, to = 1)),
+        List(appends(1, append(offset = 1, seqNr = 2)), deletes(offset = 2, to = 1)),
       ),
       (
-        Nel.of(append(offset = 1, seqNr = 1, seqNrs = 2, 3), delete(offset = 2, seqNr = 1)),
-        List(appends(1, append(offset = 1, seqNr = 1, seqNrs = 2, 3)), deletes(offset = 2, seqNr = 1)),
+        Nel.of(append(offset = 1, seqNr = 1, seqNrs = 2, 3), delete(offset = 2, to = 1)),
+        List(appends(1, append(offset = 1, seqNr = 1, seqNrs = 2, 3)), deletes(offset = 2, to = 1)),
       ),
       (
-        Nel.of(append(offset = 1, seqNr = 1), delete(offset = 2, seqNr = 1), append(offset = 3, seqNr = 2)),
-        List(appends(1, append(offset = 1, seqNr = 1)), deletes(offset = 2, seqNr = 1), appends(3, append(offset = 3, seqNr = 2))),
+        Nel.of(append(offset = 1, seqNr = 1), delete(offset = 2, to = 1), append(offset = 3, seqNr = 2)),
+        List(appends(1, append(offset = 1, seqNr = 1)), deletes(offset = 2, to = 1), appends(3, append(offset = 3, seqNr = 2))),
       ),
       (
         Nel.of(
           append(offset = 1, seqNr = 1),
-          delete(offset = 2, seqNr = 1, origin = "origin1"),
+          delete(offset = 2, to    = 1, origin = "origin1"),
           append(offset = 3, seqNr = 2),
-          delete(offset = 4, seqNr = 2, origin = "origin2"),
+          delete(offset = 4, to    = 2, origin = "origin2"),
         ),
-        List(appends(3, append(offset = 3, seqNr = 2)), deletes(offset = 4, seqNr = 2, origin = "origin1")),
+        List(appends(3, append(offset = 3, seqNr = 2)), deletes(offset = 4, to = 2, origin = "origin2")),
```

Comment:
here we lose the `origin`

Reply:
I did it on purpose, however I'm not confident was it right decision. Could you pls review the PR ignoring `origin`? Later, if the PR will be accepted - I will back to `origin`

Reply:
I am starting to lean on idea that we should just update `origin` and `version` in `metajournal` on any action, if current version is `None` - the fact that client was able to restore state from some older journal means that it is compatible with current version of kafka-journal - thus we can update.

Eventually all actively used aggregates will have them both set, though we still might have some old and stale journals without `origin` and `version` values

Honestly, I do not see, yet, good reason to aim for very first action, but I might not see some edge case, where it is important.

I think that:
* `origin` was introduced to help with manual untangling in case of split-brain
* `version` was introduced to provide support for possible future journal upgrades or compatibility checks